sentry-rails/lib/sentry/rails/tracing/abstract_subscriber.rb

```diff
@@ -14,11 +14,11 @@ def unsubscribe!
 
           def subscribe_to_event(event_name)
             if ::Rails.version.to_i == 5
-              ActiveSupport::Notifications.subscribe(event_name) do |_, start, finish, _, payload|
+              ActiveSupport::Notifications.subscribe(event_name) do |*args|
                 next unless Tracing.get_current_transaction
 
-                duration = finish.to_f - start.to_f
-                yield(event_name, duration, payload)
+                event = ActiveSupport::Notifications::Event.new(*args)
```

Comment:
I followed the usage in the official comments.

https://github.com/rails/rails/blob/v5.0.0/activesupport/lib/active_support/notifications/instrumenter.rb#L66-L77